hw/pci-bridge/simbridge_utils.h

```diff
@@ -18,6 +18,7 @@ typedef enum simmsgtype_e {
     SIMMSG_SYNC_REQ,
     SIMMSG_SYNC_ACK,
     SIMMSG_SYNC_REL,
+    SIMMSG_STEP_TIMER_WHEEL,
```

Comment:
I don't think qemu should be aware of TIMER_WHEELs.  Can we make this just a generic "advance time" message and let the server side turn that into a STEP_TIMER_WHEEL?

Reply:
makes sense, I can push conversion logic lower